pysat/tests/test_instruments.py

```diff
@@ -34,6 +35,18 @@ def safe_data_dir():
     return saved_path
 
 
+def remove_files(inst):
+    # remove any files
+    dir = inst.files.data_path
```

Comment:
dir is a special word. Please avoid.

Reply:
We may also want check in here that there aren't many files before we delete. I'd hate to somehow accidentally delete a bunch of good data.

Reply:
Since we are in a temp directory, this should be fine. Maybe a check for the modified instrument file list before a delete?

Reply:
I think I can drop the target and just use:
```
for the_file in os.listdir(inst.files.files.values):
```
Since we've swapped directories, this list is reset after the test data is downloaded.

Reply:
Maybe if the number of files are less than some level?  TIMED/SABER downloads 15 files as part of the test.

Reply:
I was thinking of something like that, perhaps 20 files or less? This check could lead to its own problems though.

 I agree with you that being in a temp directory that things should be fine. Can't help but be extra wary when deleting things in code that will run on somebody else's system.

Reply:
Added a 20 file check.  If 20 or more, no deletions occur and a warning message is issued.

Reply:
@jklenzing was the 'dir' variable name changed?